app/src/androidTest/java/ingsw/group1/findmyphone/contacts/SMSContactManagerTest.java

```diff
@@ -56,7 +56,7 @@ public void addContact() {
     public void addContact1() {
         contactManager.addContact(peerTest, CONTACT_VALID_NAME);
 
-        assertTrue(contactManager.containsPeer(peerTest));
+        Assert.assertTrue(contactManager.containsPeer(peerTest));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertTrue' due to existing static import 'org.junit.Assert.assertTrue'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5114278)